be/src/util/threadpool.h

```diff
@@ -64,6 +64,8 @@ class Thread;
 class ThreadPool;
 class ThreadPoolToken;
 
+#define THREAD_POOL_DEFAULT_IDLE_TIMEOUT_MS 500
```

Comment:
should be a constant variable instead of macro

Reply:
done